packages/circuit-ui/components/CalendarTagTwoStep/CalendarTagTwoStep.tsx

```diff
@@ -138,39 +138,52 @@ export default class CalendarTagTwoStep extends Component {
   };
 
   render() {
-    const { clearText, confirmText, onDatesRangeChange, ...props } = this.props;
+    const {
+      clearButtonLabel,
+      confirmButtonLabel,
+      onDatesRangeChange,
+      ...props
+    } = this.props;
     const { focusedInput, startDate, endDate } = this.state;
     const isOpen = focusedInput !== null;
-    const isFilled = !!(startDate && endDate);
+    const isFilled = Boolean(startDate && endDate);
 
     return (
       <div {...props}>
         <Tag
           selected={isOpen || isFilled}
-          ref={this.handleTagRef}
+          ref={this.tagRef}
           onClick={this.handleTagClick}
         >
           {this.getDateRangePreview()}
         </Tag>
         {isOpen && (
           <CalendarWrapper>
+            {/* @ts-expect-error This worked before the component was converted to TypeScript */}
```

Comment:
I didn't bother fixing the types here since the component will soon be replaced.